repos/keg-core/core/base/components/meta/versionDisplay.js

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback, useMemo } from 'react'
+import { Button } from '@keg-hub/keg-components'
+import { Linking } from 'react-native'
+
+/**
+ * @param {string} homepage 
+ * @param {string} version 
+ * @return {Function} - callback for opening the release notes page for the 
+ * tap version
+ */
+const useOpenHomepageCallback = (homepage, version) => useCallback(
+  () => {
+    if (!homepage) return
+
+    const versionEndpoint = version ? `/v${version}` : ''
+    const uri = `${homepage}/releases/${versionEndpoint}`
+
+    Linking
+      .canOpenURL(uri)
+      .then(supported => supported 
+        ? Linking.openURL(uri)
```

Comment:
Is there a way to make this open in a new tab?
When testing, it used the same tag :(

Reply:
changed to a link component with `target={'_blank'}`, so this now works